### DISCUSSION TITLE
Z2M after Update Not stable

### DISCUSSION THREAD

**Discussion by Almighty1985** [Koenkk/zigbee2mqtt#17925] [General]
Since updating to 1.31.0 from 1.30.4 every few hours the CPU and RAM usage of the addon goes to 50% and all my devices are "unavaiable". After manually restarting Z2M everything is back to normal for a few hours, until the usage goes up another time.

The logs looked good (no error message, but also new status update from the devices). Am I the only one with this problem?

**Comment by lux73** [comment]
duplicate -> https://github.com/Koenkk/zigbee2mqtt/discussions/17852#discussioncomment-6086893

**Reply by DanielB1982** [reply]
Same problem, random restarts of zigbee2mqtt. Raspberry pi 4 and conbee 2 Stick. Currently no time and local access to the Hardware to check the problem

**Comment by ginkel** [comment]
I am also seeing this since upgrading from 1.30.4 to 1.31.2. Starting yesterday evening all devices have started to go offline ("failed to ping"). Will downgrade for now.

**Comment by Shred99** [comment]
I haven't had a problem with Z2M going offline yet, but since upgrading my Dockerised install from 1.30.3 to 1.31.2 a few days ago, I've noticed that Zabbix is showing that "free memory" on my Raspberry Pi is steadily decreasing - as if there is a memory leak.  It's possible that something else could be causing it: I haven't investigated yet.

I only have 28 devices, so mine may take longer to experience problems than others.

![image0](https://github.com/Koenkk/zigbee2mqtt/assets/9124236/acd8355a-0f5c-4849-8a8e-2981aea17df7)
